qa-pipelines/tasks/cf-upgrade.sh

```diff
@@ -52,33 +52,36 @@ set_uaa_sizing_params # Adds uaa sizing params to HELM_PARAMS
 echo UAA customization ...
 echo "${HELM_PARAMS[@]}" | sed 's/kube\.registry\.password=[^[:space:]]*/kube.registry.password=<REDACTED>/g'
 
-helm upgrade uaa ${CAP_DIRECTORY}/helm/uaa/ \
-    --namespace "${UAA_NAMESPACE}" \
-    --recreate-pods \
-    --timeout 3600 \
-    --wait \
-    "${HELM_PARAMS[@]}"
-
-# Wait for UAA release
-wait_for_release uaa
+if [ "${EMBEDDED_UAA:-false}" != "true" ]; then
```

Comment:
[[ ]] for consistency

Reply:
Fixed in 86a2d8d13026096aad7fecfcb1d52bb81fe4f16e.